### ISSUE TITLE
Add support for an expand option?

### ISSUE THREAD

**Issue by micahblu** [yoniholmes/grunt-text-replace#25]
It be very nice to maintain the directory structure from src wildcard matches.

For example let's say I have:

``` javascript
replace: {
  html: {
    src: ['src/**/*.html'],
    dest: 'dist/'
    // the rest
  }
}
```

In this case the dist gets a flattened version of the src files, losing directories and possibly overwriting files, for example a directory with index.html would overwrite the root index.html. It'd be nice to have an expand option that would build the directories as needed.

**Comment by fantapop** [comment]
I agree this would be nice.  A work around would be to copy the tree to your new location and then run the replace on the files in place.

**Comment by micahblu** [comment]
I actually have a detached fork where I implemented the feature: https://github.com/micahblu/grunt-replace-text

**Comment by fantapop** [comment]
okay, no pull request?

**Comment by micahblu** [comment]
I did, but he didn't reply in time and I needed an updated git repo for dev deployments at work

**Comment by micahblu** [comment]
I'll do another pull request, I believe he did eventually reply.

**Comment by fantapop** [comment]
That would be appreciated. Thank you!

**Comment by micahblu** [comment]
I've submitted a pull request that should do the trick
